### DISCUSSION TITLE
Edge Functions Locations

### DISCUSSION THREAD

**Discussion by thebergamo** [supabase/supabase#12385] [Questions] [answered]
Hi Guys!
Is it correct to assume that the Edge Function locations are the same available for the project selection or there is an extended list of locations specific for edge functions?

Currently I'm making my bachelor thesis and one of the aspects I'm analysing is the availability of the Edge Functions providers, I understand that Supabase can be self-hosted, but at this point I'm analysing only hosted options.

Thank you very much for the response in advance and just would like to say that Supabase is an amazing product o/

**Comment by GaryAustin1** [comment, marked as answer]
The edge functions actually use the Deno service so would be their network.   I do not know if SB could pick or choose which ones.

**Reply by thebergamo** [reply]
I saw they use Deno as their platform, but didn't know it was also using Deno Deploy as the platform for running the Edge Functions. Is there any place where it is stated?

Thank you @GaryAustin1 for your answer :)

**Reply by GaryAustin1** [reply]
https://deno.com/blog/supabase-functions-on-deno-deploy

**Reply by GaryAustin1** [reply]
https://supabase.com/blog/supabase-edge-functions
See "The Supa-Deno Platform"
